### PR TITLE
Bootstrap docker image for OneAgent

### PR DESCRIPTION
**Pull request by aloismayr**
This PR covers a new integration with the bootstrap image for Dynatrace OneAgent.

### CODE REVIEW

File: repo/packages/D/dynatrace/1/package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "packagingVersion": "3.0",
+    "name": "dynatrace",
+    "version": "latest",
```
Comment (ryadav88) [inline repo/packages/D/dynatrace/1/package.json]: `version` should be `baseTechVersion-integrationVersion`

File: repo/packages/D/dynatrace/1/package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "framework": false,
+    "licenses": [
+        {
+            "name": "MIT License",
+            "url": "https://github.com/Dynatrace/Dynatrace-Docker/blob/master/LICENSE"
+        }
+    ],
+    "postInstallNotes": "Dynatrace has been installed on DC/OS Mesos agents.\nLog in to your Dynatrace account at https://login.dynatrace.com and monitor your DC/OS cluster. You will shorty see your cluster and container insights in Dynatrace.",
+    "postUninstallNotes": "Dynatrace has been uninstalled from your Mesos agents and will no longer run.",
+    "preInstallNotes": "Pass your environment ID (dynatrace.environment), token (dynatrace.token), and connection endpoint (dynatrace.server) as package options.\nPlease note the connection endpoint (dynatrace.server) of Dynatrace SaaS is https://{replace_with_your_environmentID}.live.dynatrace.com.\nIn case you are running Dynatrace Managed you can find the Managed endpoint in your UI.\nAlso pass the number of Mesos agents as an option (dynatrace.instances) and Marathon will make sure that the Dynatrace OneAgent runs on every Mesos agent.\nBy default Marathon will allcoate 0.1 CPU and 400m memory per Dynatrace OneAgent task.",
```
Comment (ryadav88) [inline repo/packages/D/dynatrace/1/package.json]: Add this line at the beginning of preInstallNotes: `This DC/OS Service is currently in preview. There may be bugs, incomplete features, incorrect documentation, or other discrepancies.`

File: repo/packages/D/dynatrace/1/resource.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "images": {
+    "icon-small": "https://assets.dynatrace.com/global/resources/Signet_Logo_RGB_CP_48x48px.png",
+    "icon-medium": "https://assets.dynatrace.com/global/resources/Signet_Logo_RGB_CP_96x96px.png",
+    "icon-large": "https://assets.dynatrace.com/global/resources/Signet_Logo_RGB_CP_256x256px.png"
+  },
+  "assets": {
+    "container": {
+      "docker": {
+        "agent": "dynatrace/oneagent:latest"
```
Comment (ryadav88) [inline repo/packages/D/dynatrace/1/resource.json]: please use a tagged version of an image

Comment (aloismayr) [reply repo/packages/D/dynatrace/1/resource.json]: Would `dynatrace/oneagent:mesos` work for you?

File: repo/packages/D/dynatrace/1/package.json (hunk 2 above)
Comment (aloismayr) [reply repo/packages/D/dynatrace/1/package.json]: Why do you want us to add this? What would be needed to merge the PR without that message?

File: repo/packages/D/dynatrace/1/package.json (hunk 1 above)
Comment (aloismayr) [reply repo/packages/D/dynatrace/1/package.json]: sounds good

Comment (aloismayr) [reply repo/packages/D/dynatrace/1/package.json]: done

File: repo/packages/D/dynatrace/1/package.json (hunk 2 above)
Comment (aloismayr) [reply repo/packages/D/dynatrace/1/package.json]: done

File: repo/packages/D/dynatrace/1/resource.json (hunk 3 above)
Comment (aloismayr) [reply repo/packages/D/dynatrace/1/resource.json]: done

**Pull request merged**